plugins/gradle/example-server/gradle/wrapper/gradle-wrapper.properties

```diff
@@ -0,0 +1,5 @@
+distributionBase=GRADLE_USER_HOME
+distributionPath=wrapper/dists
+distributionUrl=https\://services.gradle.org/distributions/gradle-7.1-bin.zip
+zipStoreBase=GRADLE_USER_HOME
+zipStorePath=wrapper/dists
```

Comment:
@aldib could you please remove gradle wrapper files from example-server?
You can build any subdirectory using:
`./gradlew -p plugins/gradle/example-server clean build --warning-mode all`

Reply:
Removed. Thank you